src/main/java/com/revature/sfadapter/services/SFAuthService.java

```diff
@@ -1,86 +1,93 @@
 package com.revature.sfadapter.services;
 
-import com.revature.sfadapter.data.services.DaoService;
+
 import com.revature.sfadapter.util.SFWSAccessObject;
 import org.apache.log4j.Logger;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
-import org.springframework.util.Assert;
-import org.springframework.context.annotation.Bean;
-import org.springframework.http.HttpEntity;
 import org.springframework.stereotype.Service;
+import org.springframework.util.Assert;
 import org.springframework.util.LinkedMultiValueMap;
 import org.springframework.util.MultiValueMap;
-import org.springframework.util.SystemPropertyUtils;
-import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.client.RestTemplate;
 
-import javax.print.attribute.standard.Media;
-import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.net.URLEncoder;
-import java.util.*;
+import java.util.Arrays;
+import java.util.Map;
 
 @Service
 public class SFAuthService {
 
-	private DaoService<SFWSAccessObject, String> tokenService;
+	private static final String clientId;
+	private static final String clientSecret;
+	private static final String authorizationUrl;
+	private static final String myCallbackUrl;
+
+	static{
+		Map<String, String> env = System.getenv();
+		clientId = env.get("CONSUMER_KEY");
+		clientSecret = env.get("CONSUMER_SECRET");
+		authorizationUrl = env.get("SF_AUTH_SERVER");		// *.saleforce.com/
+		myCallbackUrl = env.get("CALLBACK_URL");
+	}
 
 	private static final Logger LOGGER = Logger.getLogger(SFAuthService.class);
-	@Autowired
-	public void setTokenService(DaoService<SFWSAccessObject, String> tokenService){
-		Assert.notNull(tokenService);
-		this.tokenService = tokenService;
+
+	private String buildAuthorizationUrl(String endpoint, String clientRedirect){
+
+		//StringBuilder url = new StringBuilder(String.format("%s?response_type=code", authorizationUrl));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revatureops.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)